tests/src/test/scala/mainecoon/tests/autoFunctorTests.scala

```diff
@@ -18,55 +18,39 @@ package mainecoon
 package tests
 
 
-import cats.laws.discipline.SerializableTests
-import cats.laws.discipline.FunctorTests
-import autoFunctorTests._
 import cats.Functor
+import cats.instances.function._
 import cats.kernel.Eq
-import org.scalacheck.{Arbitrary, Cogen}
+import cats.kernel.instances.string._
+import cats.kernel.instances.tuple._
 import cats.laws.discipline.eq._
+import cats.laws.discipline.{FunctorTests, SerializableTests}
+import cats.syntax.arrow._
+import mainecoon.tests.autoFunctorTests._
+import org.scalacheck.{Arbitrary, Cogen}
 
 class autoFunctorTests extends MainecoonTestSuite {
 
-  checkAll("SimpleAlg[Option]", FunctorTests[SimpleAlg].functor[Float, String, Int])
-  checkAll("Functor[SimpleAlg]", SerializableTests.serializable(Functor[SimpleAlg]))
-
-  test("non effect method correctly delegated") {
-    val doubleAlg = AlgWithNonEffectMethodFloat.map(_.toDouble)
-    doubleAlg.bar("big") should be("big")
-    doubleAlg.foo("big") should be(3d)
-  }
+  checkAll("SimpleAlg[Option]", FunctorTests[TestAlgebra].functor[Long, String, Int])
+  checkAll("Functor[SimpleAlg]", SerializableTests.serializable(Functor[TestAlgebra]))
```

Comment:
since you replaced `SimpleAlg` with `TestAlgebra`, would you change the text here as well?